.Tests/XUnit.Tests/GameServerAttributeTest.cs

```diff
@@ -4,6 +4,7 @@
 
 namespace XUnit.Tests;
 
+#if false
 public class GameServerAttributeTest
```

Comment:
Use Skip for xUnit tests - we will see which tests was skipped + you can also add message

Reply:
I remove all code which this class was testing. It is just compiler errors now. i should just delete the whole file tbh.